types/stream-chat/api-response-tests/index.js

```diff
@@ -42,6 +46,8 @@ const executables = [
 	{ f: rg.getReactions, type: 'GetReactionsAPIResponse' },
 	{ f: rg.updateChannel, type: 'UpdateChannelAPIResponse' },
 	{ f: rg.deleteChannel, type: 'DeleteChannelAPIResponse' },
+	/** Keeping truncate quite for now. Needs some fixes on backend regardind commands array in response */
```

Comment:
typo and could you explain what is needed to be fixed?

Reply:
<img width="825" alt="Screenshot 2020-04-09 at 18 03 12" src="https://user-images.githubusercontent.com/11586388/78919053-a37dd200-7a91-11ea-90dd-5253f77c6a27.png">

commands is different in truncateChannel response. It should be array of objects :) 

Marcelo has fixed it. It will be deployed by tomorrow

Reply:
yes, I saw the discussion with Mike and it's good to have a reference. I think this comment is enough, no need write in source since it will be updated soon.